strings/salesman_replacement.json

```diff
@@ -0,0 +1,49 @@
+{
+
+    "salesman": {
+        "shuttle": "cungadero",
+        "clam": "pipis",
+        "clams": "pipis",
+        "egg": "pipis",
+        "eggs": "pipis",
```

Comment:
these are really uncommon terminology

Reply:
Eggs are pretty common though, especially as they're one of the few roundstart food items the chef constantly has. Clams are rare, but somebody suggested i add it in because pipis are like clams.

Reply:
it's because pipis is a type of clam (that has been struggling with page vandalism for some reason lately)

Reply:
I was the one who suggested this to crit so i'll answer this:
its more for completion's sake, in the source material [pipis] are memed to the Nth dimension, and through a special dialogue its revealed that the [Pipis] are an invasive species of freshwater **clams** also they look like eggs.